scenarios/centos-9/edpm_ci.yml

```diff
@@ -4,16 +4,7 @@ cifmw_installyamls_repos: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-o
 cifmw_install_yamls_vars:
   DATAPLANE_REPO: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/dataplane-operator"
   STORAGE_CLASS: crc-csi-hostpath-provisioner
-
-cifmw_use_crc: true
-cifmw_use_libvirt: true
-cifmw_rhol_crc_use_installyamls: true
```

Comment:
We need to add this var back to the file otherwise 
```
2023-08-22 06:04:12.809925 | controller | TASK [Attach default network to CRC name=install_yamls_makes, tasks_from=make_crc_attach_default_interface.yml] ***
2023-08-22 06:04:12.809936 | controller | Tuesday 22 August 2023  06:04:12 -0400 (0:00:00.040)       0:00:58.397 ********
2023-08-22 06:04:12.851114 | controller | skipping: [localhost]
```
gets skipped and it breaks `make openstack` step with f[ollowing error.](https://logserver.rdoproject.org/93/49693/1/check/periodic-podified-edpm-deployment-antelope-ocp-crc-1cs9/327fed8/ci-framework-data/logs/ci_make_001_run_openstack.log)
```
error: timed out waiting for the condition on nodenetworkconfigurationpolicies/enp6s0-crc-74q6p-master-0
```
By adding it back, it will fix the issue.

Reply:
I think that `cifmw_extras` is getting overridden, cause the nested scenario is passed in the base job.